courses/factories.py

```diff
@@ -1,13 +1,18 @@
-# pylint: disable=missing-docstring
+"""Factories for testing"""
 import factory
 from factory.django import DjangoModelFactory
 from .models import Module, Course
 
+
 class CourseFactory(DjangoModelFactory):
-    class Meta:
+    """Factory for Course"""
```

Comment:
It can be helpful for documentation in general to have this pylint rule. In this particular case it probably doesn't add value but I don't see the harm in having them there anyway